services/staging/tests/test_staging.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 CS Group
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test staging module."""
+
+import pytest
+from starlette.status import HTTP_200_OK
+
+
+@pytest.mark.unit
+def test_ping(staging_client):
+    """Test status."""
+    response = staging_client.get("/_mgmt/ping")
+    assert response.status_code == HTTP_200_OK
+    assert response.json() == {"status": "healthy"}
```

Comment:
You have an error in your pytests, it's a detail it's just the response of this health endpoint that is different from the other services. Your staging service returns the string "Healthy" while the other services return the json {"status": "healthy"}. I guess it would be best if you also return this json so the configuration will be the same in kubernetes.

Reply:
Fixed, liveness probe should only be checked if return is 200OK.